policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_partial_credit_limit.yaml

```
@@ -0,0 +1,10 @@
+description: Ohio Child and Dependent Care credit partial credit threshold
```

Not sure if this is actually what it does, but please phrase descriptions this way
```suggestion
description: Ohio starts phasing out its Child and Dependent Care credit at this income level.
```